egs/wsj/s5/utils/validate_lang.pl

```diff
@@ -760,6 +760,8 @@ sub check_summation {
       if (! defined $is_disambig{$phone}) {
         if ($phone == "<<eos>>") {
           $state = "eos";
+        } elsif ($phone == 0) {
+          $exit = 1; print "--> ERROR: unexpected phone sequence=$phoneseq, wordseq=$wordseq\n"; last;
```

Comment:
cool, but can  you change the == to eq?
I'm a little confused why the diff here seems to be against the older version of this code, not the new (and  presumably broken?) one.